x-pack/elastic-agent/pkg/reporter/log/format.go

```diff
@@ -21,7 +21,7 @@ const (
 
 const (
 	// e.g "2006-01-02T15:04:05: type: 'STATE': event type: 'STARTING' message: Application 'filebeat' is starting."
```

Comment:
Should update the comment to match the change.

Reply:
updated